example/10-secret-internal-domain.yaml

```
@@ -1,14 +1,11 @@
 ---
+# Secret is referenced by the Seed's spec.dns.internal field.
+# Please see https://gardener.cloud/docs/gardener/deployment/setup_gardener/#dns-setup-for-internal-external-domains
 apiVersion: v1
 kind: Secret
 metadata:
   name: internal-domain-example-com
```

Cosmetic suggestion:

```suggestion
  name: internal-domain-secret
```

so that the name in the example Seed spec match the name in the example internal domain secret.